AssetInformationListener.Tests/E2ETests/Steps/DoSomethingUseCaseSteps.cs

```diff
@@ -0,0 +1,80 @@
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+using Amazon.Lambda.TestUtilities;
+using AutoFixture;
+using AssetInformationListener.Boundary;
+using AssetInformationListener.Infrastructure;
+using AssetInformationListener.Infrastructure.Exceptions;
+using FluentAssertions;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AssetInformationListener.Tests.E2ETests.Steps
+{
+    public class DoSomethingUseCaseSteps : BaseSteps
```

Comment:
I think we should change the file name to something more descriptive

Reply:
It will be when the next ticket is done. This is just setting up the application shell.